elements/pf-jump-links/pf-jump-links.ts

```diff
@@ -100,23 +101,26 @@ export class PfJumpLinks extends LitElement {
 
   render() {
     return html`
-      <nav id="container">${this.expandable ? html`
+      <nav id="container" @slotchange="${this.#updateItems}">${this.expandable ? html`
         <details ?open="${this.expanded}" @toggle="${this.#onToggle}">
           <summary>
             <pf-icon icon="chevron-right"></pf-icon>
             <span id="label">${this.label}</span>
           </summary>
-          <slot role="listbox" @slotchange="${this.#onSlotchange}"></slot>
+          <slot role="listbox" @slotchange="${this.#updateItems}"></slot>
         </details>` : html`
         <span id="label">${this.label}</span>
-        <slot role="listbox" @slotchange="${this.#onSlotchange}"></slot>`}
+        <slot role="listbox" @slotchange="${this.#updateItems}"></slot>`}
       </nav>
     `;
   }
 
-  #onSlotchange() {
-    this.#items = this._items;
-    const items = this.#items?.map(item=>item.link);
+  #updateItems() {
```

Comment:
we can do all the work here, privately, instead of exposing new APIs on the child elements.

In my brief testing, this behaved the same as before I made this change. I'll leave this comment open to block auto-merge until you can review the DP